geometry_msgs/Vector3 robot_pos
ParticleInfo[] particles
